<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeprice</class>
 <widget class="QDialog" name="changeprice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="srButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>单人间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="brButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>大床房</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>双人间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="prButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总统套房</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>新价格</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
